Gruntfile.js

```diff
@@ -165,6 +165,21 @@ module.exports = function (grunt) {
                 src: 'src/main/webapp/js/min/script.js',
                 dest: 'src/main/webapp/js/min/script.clean.js'
             }
+        },
+        protractor : {
+            options : {
+                keepAlive : true,
+                noColor : false,
```

Comment:
If true, protractor will not use colors in its output.
I guess 'true' is more suitable